fcrepo-indexing-solr/src/main/java/org/fcrepo/camel/indexing/solr/SolrRouter.java

```diff
@@ -94,7 +95,7 @@ public void configure() throws Exception {
             .filter(not(or(header(IDENTIFIER).startsWith(simple("{{audit.container}}/")),
                     header(IDENTIFIER).isEqualTo(simple("{{audit.container}}")))))
             .to("fcrepo:{{fcrepo.baseUrl}}?preferOmit=PreferContainment")
-            .setHeader(FCREPO_TRANSFORM).xpath(hasIndexingTransform, String.class, ns)
+            .setHeader(FCREPO_TRANSFORM).xpath(hasIndexingTransformation, String.class, ns)
```

Comment:
That is to say, the property we're using here is completely arbitrary. It could just as well be `ex:foo`, but we're using the convention established in the fcrepo-message-consumer.